metals/src/main/scala/scala/meta/internal/troubleshoot/ScalaProblem.scala

```diff
@@ -0,0 +1,75 @@
+package scala.meta.internal.troubleshoot
+
+import scala.meta.internal.metals.BuildInfo
+import scala.meta.internal.metals.ScalaVersions
+
+sealed abstract class ScalaProblem {
```

Comment:
More of just a curiosity, but I notice in a lot of Scalameta projects we prefer `abstract class` over `trait`. What's the main reason we do that?

Reply:
Haha, no idea, I didn't even realize I created an abstract class. :sweat_smile: